packages/devtools_app/lib/src/banner_messages.dart

```diff
@@ -254,6 +254,50 @@ Relaunch your application with the '--profile' argument, or ''',
   }
 }
 
+class ProviderUnknownErrorBanner {
```

Comment:
both these messages are specific to your screens so put them in the directory of the screen rather than the generic messages file.